recipes/symengine/all/test_package/conanfile.py

```diff
@@ -1,18 +1,26 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import CMake, cmake_layout
 import os
 
 
 class TestPackageConan(ConanFile):
-    settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package_multi"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
+    test_type = "explicit"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
 
     def build(self):
         cmake = CMake(self)
-        cmake.definitions["CMAKE_CXX_STANDARD"] = "11"
-        cmake.configure()
+        cmake.configure(variables={"CMAKE_CXX_STANDARD": "11"})
```

Comment:
This should be passed in at the toolchain :)

Reply:
Done.